linetools/spectra/lsf.py

```diff
@@ -213,6 +230,143 @@ def load_COS_data(self):
         
         return pixel_scale, data
 
+    def load_STIS_data(self):
+        """Load the right data according to `instr_config` for HST/STIS
+        instrument"""
+
+        # define pixel scales; values obtained from STScI
+        # these values must be consistent with the given LSFs
+        pixel_scale_dict = {
+            'G140L': 0.60 * u.AA,
+            'G140M': 0.05 * u.AA,
+            'G230L': 1.58 * u.AA,
+            'G230M': 0.09 * u.AA,
+            'E140M': 'lambda/91700',
+            'E140H': 'lambda/228000',
+            'E230M': 'lambda/60000',
+            'E230H': 'lambda/228000',
+            'G230LB': 1.35 * u.AA,
+            'G230MB': 0.15 * u.AA,
+            'G430L': 2.73 * u.AA,
+            'G430M': 0.28 * u.AA,
+            'G750L': 4.92 * u.AA,
+            'G750M': 0.56 * u.AA
+        }
+        # define channel based on grating name
+        channel_dict = {
+            'G140L': 'FUV-MAMA',
+            'G140M': 'FUV-MAMA',
+            'G230L': 'NUV-MAMA',
+            'G230M': 'NUV_MAMA',
+            'E140M': 'FUV-MAMA',
+            'E140H': 'FUV-MAMA',
+            'E230M': 'NUV-MAMA',
+            'E230H': 'NUV-MAMA',
+            'G230LB': 'CCD',
+            'G230MB': 'CCD',
+            'G430L': 'CCD',
+            'G430M': 'CCD',
+            'G750L': 'CCD',
+            'G750M': 'CCD'
+        }
+        # also need slits
+        available_slits = {
+            'G140L': ['52x0.1', '52x0.2', '52x0.5', '52x2.0'],
```

Comment:
This is certainly a bug and I know how to fix it. Will work on it during the next days... Thanks for spotting it!